modules/installation-osp-local-disk-deployment.adoc

```diff
@@ -0,0 +1,296 @@
+// Module included in the following assemblies:
+//
+// * installing_openstack/deploying-openstack-with-rootVolume-etcd-on-local-disk.adoc
+
+:_mod-docs-content-type: Procedure
+[id="installation-osp-local-disk-deployment_{context}"]
+= Deploying {rh-openstack} on local disk
+
+.Prerequisites
+
+* You have an OpenStack cloud with a working Cinder, and it has at least 75 GB of available storage to accommodate 3 root volumes for the OpenShift control plane.
+
+* The OpenStack cloud must be deployed with Nova ephemeral storage that uses a local storage backend and not RBD.
+
+
+.Procedure
+
+[WARNING]
+====
+This procedure can testing etcd on local disk and should not be used on production cluster because in certain cases complete loss of the control plane can occur. For more information, see "Overview of backup and restore operation" under "Backup and restore".
+====
+
+. Create a Nova flavor for the control plane with 10 GB of ephemeral disk by running the following command:
```

Comment:
with at least 10GB; I wonder if we have other manuals on sizing a disk for etcd, for our enterprise customers with large clusters.

Reply:
BTW the command is an example, RAM & CPU or other attributes can be changes. The point is to show the `--ephemeral` argument.